client/my-sites/stats/stats-upsell/style.scss

```diff
@@ -40,34 +38,40 @@
 	}
 
 	.stats-upsell__left {
+		padding: 40px;
 		background: var(--studio-white);
 		gap: 24px;
-		@include break-medium {
+		@include break-wide {
 			flex: 1;
 		}
 	}
 
 	.stats-upsell__right {
-
 		gap: 8px;
-		@include break-medium {
+		@include break-wide {
 			flex: 1;
 		}
+
+		img {
+			width: 100%;
```

Comment:
```suggestion
			height: 100%;
```
It may be fittable using full height and auto-adjusting the width because the image is landscape.

#### Before
<img width="992" alt="截圖 2024-12-02 晚上11 25 59" src="https://github.com/user-attachments/assets/538c4a7a-9496-4c32-a44a-22f004757a0d">